fmf.spec

```diff
@@ -1,16 +1,19 @@
-Name: fmf
-Version: 1.4.1
-Release: 1%{?dist}
+Name:          fmf
+Version:       0.0.0
+Release:       1%{?dist}
 
-Summary: Flexible Metadata Format
-License: GPLv2+
-BuildArch: noarch
+Summary:       Flexible Metadata Format
+License:       GPL-2.0-or-later
+BuildArch:     noarch
 
-URL: https://github.com/psss/fmf
-Source0: https://github.com/psss/fmf/releases/download/%{version}/fmf-%{version}.tar.gz
+URL:           https://github.com/teemtee/fmf
+Source:        %{pypi_source fmf}
 
 # Main fmf package requires the Python module
-Requires: python%{python3_pkgversion}-%{name} == %{version}-%{release}
+BuildRequires: python3-devel
+BuildRequires: python3dist(docutils)
+BuildRequires: git-core
+Requires:      python3-fmf == %{version}-%{release}
```

Comment:
I would suggest to do the same as in https://github.com/teemtee/tmt/blob/main/tmt.spec, i.e. `%py_provides    python3-fmf`

Reply:
This was discussed in other, now resolved thread.  
@psss Would you agree that we should build `fmf` package, which would also provide `python3-fmf`, in the same fashion as tmt?

Reply:
Do you remember why there wasn't a `Obsoletes: python3-tmt`?

Reply:
hmm, where? This? https://github.com/teemtee/tmt/blob/0af72a033fab8b385ae3a7da05568b3e3dc9593e/tmt.spec#L16

Reply:
Oh, didn't see that. :+1: then